exercises/spiral-matrix/description.md

```diff
@@ -0,0 +1,22 @@
+Given a number, return a matrix of numbers in spiral order.
```

Comment:
I reworded it, trying to keep it as short as possible and avoid repeating the next paragraph. I hope it is better now.